scripts/lc_classifiers_letters_only.json

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
What is the provenance of this file (and the other JSON file)?

Reply:
We [parse these .pdf files](http://www.loc.gov/catdir/cpso/lcco) with a variant of https://github.com/internetarchive/openlibrary/issues/3290#issuecomment-607726981  (parsing code is MIT License) That is the linkage to that issue.

This is a WIP Draft because there are trade-offs to consider between looking something up in a dict vs. embedding it in every record.